public/orbs-public-beta.md

```diff
@@ -99,134 +102,134 @@ To install Polygon run
+* `<ethereum endpoint url>` - a url to a working and synced Ethereum node. 
+* `<ssh source cird block>` - One or more CIDR blocks which will be granted access to ssh from into the node. You will still need the public key to connect - it is required only in cases of troubleshooting. The format is standard CIDR. Any IP not in the range will not be able to SSH to the node, even if it has the SSH key file. To allow ssh access from any ip use `"incomingSshCidrBlocks": ["0.0.0.0/0"],` 
 
-Other parameters (no need to change them):
 
-The `cachePath` configuration tells Polygon where to store the terraform installation meta-data created during the deploy stage. It is required in cases where you wish to remove the node from AWS. You should store these files and back them up so you can run maintenance if required.
+## Update Node configuration
 
-The `awsProfile` configuration can be changed if you are using multiple AWS configurations and want a specific one to be applied.
+__Any modification to your configuration file MUST be done while the node is DOWN.__
 
-## Some warning as to updating your node configuration JSON file
-While the configuration is quite easily changeable, please do remember that any modification to your configuration file MUST be done while the node is DOWN.
-For example: if you decide you want to set a backend syncing using the Terraform state syncing to S3 (Just as an example). 
-You should do the following to perform the change:
+To update your node configuration
 * run polygon destroy
 * update your configuration JSON file to reflect your changes (for example add `"backend": true`)
 * run polygon create
 
 ### Run Polygon CLI to deploy the node
 
-To avoid having the orbs node private key as part of your command history, we recommend creating a file called `orbs-private-key.txt` and put the orbs node private key inside it, __without the leading 0x__.
-That key was generated by the key generator and should be in a hex string of size 64 characters, like `f5f83Ee70a85fFF2exxxxxxxxxxxxxxxxxxxxxxxxxxx334932F34C8D629165Ed`.
+Temporarily store the Orbs Node private key in a file `<orbs private key file>` __without the leading 0x__.
+Note, the private key should be a string of 64 characters, e.g. `f5f83Ee70a85fFF2exxxxxxxxxxxxxxxxxxxxxxxxxxx334932F34C8D629165Ed`.
 
 To provision the resources required for the node:
 
-    polygon create -f orbs-node-beta.json --orbs-private-key $(cat path/to/orbs-private-key.txt)
+    polygon create -f orbs-node-beta.json --orbs-private-key $(cat <orbs private key file>)
+
+After ensuring the private key is stored in a secure place, delete the temporary private key file `<orbs private key file>`
 
-Terraform files corresponding to nodes can be found in the folder defined in `cachePath` and should be backed up.
+During creation of the node, Terraform cache files will be created in the folder defined in `cachePath` and should be backed up to allow proper cleanup at a later stage.
```

Comment:
1. Is there a section here on how to properly destroy?
2. Would have been convenient if the creation would create a zip file with the outputs that need to be backed up (with the exception of credentials), and then the destroy command can take the zip file as an argument.